psi4/driver/driver.py

```diff
@@ -2062,6 +2062,8 @@ def molden(wfn, filename=None, density_a=None, density_b=None, dovirtual=None):
         mw = core.MoldenWriter(wfn)
         mw.write(filename, wfn.Ca(), wfn.Cb(), wfn.epsilon_a(), wfn.epsilon_b(), occa, occb, dovirt)
 
+def tdscf(name=None, **kwargs):
+    return proc.run_tdscf_energy(name = name,**kwargs)
```

Comment:
in my notions, this would become the below and `run_tdscf_excitations` is a fn that live in `proc.py` and has contents https://github.com/psi4/psi4/pull/1885/files#diff-dbf22cd80d16f6d8ef8250145a6d91bfR2709-R2732 .

```
def tdscf(wfn, **kwargs):
    return run_tdscf_excitations(wfn, **kwargs)
```

Alternately, `run_tdscf_excitations` in `proc.py` is named `tdscf` in `proc.py` and has the same contents. Then would also need to import tdscf into higher namespace for `psi4.tdscf` to work.